apps/namadillo/src/types.ts

```diff
@@ -40,7 +40,7 @@ export type AddressBalance = Record<Address, BigNumber>;
 
 export type GasConfig = {
   gasLimit: GasLimit;
-  gasPrice: GasPrice;
+  gasPriceInMinDenom: GasPrice;
```

Comment:
so, could we review this just in case? I can feel your frustration, and I share this as well. What I believe on this specifically case is that it could be the gasPrice on minDenom (for all other tokens), but for Namada this is on displayDenom, or at least this is what I understood on `gasPriceTableAtom`. If this is true, having the `InMinDenom` could lead to more confusion later. So do you think it's reasonable for this case preserve the term `gasPrice` only?

Reply:
minDenomAmount on Namada has the same value than the displayAmount :/ this is the biggest cause of confusion. So for Namada it never makes any difference, we'll never pass unam anywhere - I think we only use unam on the CLI to transfer to another chain